zScanner/DomainModel/PageDomainModel.swift

```diff
@@ -40,8 +41,10 @@ struct PageDomainModel: Equatable {
 }
 
 extension PageDomainModel {
-    init(image: UIImage, index: Int, correlationId: String) {
+    init(picture: Picture, index: Int, correlationId: String) {
         self.init(id: UUID().uuidString, index: index, correlationId: correlationId, relativePath: "")
-        self.image = image
+        
+        self.image = picture.image
+//        if let description = picture.description { self.description = description } else { self.description = "" }
```

Comment:
don't make inline `if` it just looks bad. 
you can use `let description  = picture.description ?? ""` it's called "ternary operator".

Reply:
I agree with you. Now it looks better.